packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -512,19 +510,15 @@ function SettingsSection( {
 	);
 }
 
-function DataviewsViewConfigDropdown( {
-	density,
-	setDensity,
-}: {
-	density: number;
-	setDensity: React.Dispatch< React.SetStateAction< number > >;
-} ) {
+function DataviewsViewConfigDropdown() {
 	const { view } = useContext( DataViewsContext );
 	const popoverId = useInstanceId(
 		_DataViewsViewConfig,
 		'dataviews-view-config-dropdown'
 	);
-
+	const usedLayout = VIEW_LAYOUTS.find(
```

Comment:
This is just a super minor thing, but my brain prefers `activeLayout` as a variable name :)